rabbitmq-quickstart/rabbitmq-quickstart-processor/src/main/docker/Dockerfile.native-distroless

```diff
@@ -0,0 +1,23 @@
+####
```

Comment:
These distroless files should be replaced with micro

Reply:
Not sure what this change is. Current "code.quarkus.io" generates these same docker files.

Reply:
And other quickstarts appear to have these same docker files.

Reply:
We are in the process of changing them. I've merged the PR a few days ago.

Reply:
Ah ok. Switched! I used that PR as an example.